src/controllers/QueryNotificationHandler.ts

```diff
@@ -43,7 +45,7 @@ export class QueryNotificationHandler {
     private handleCompleteNotification(): NotificationHandler<any> {
```

Comment:
May want to rename this function to handleQueryCompleteNotification to match the one in query runner (handleQueryComplete). Also it will remove ambiguity as to what is complete. Just a suggestion.